oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/searchtag/TagAutocompleteData.java

```diff
@@ -88,7 +107,29 @@ public RemoteData.Builder serialize() {
         builder.addDataStrings(tag);
         builder.addDataStrings(tagKey);
         builder.addDataStrings(tagValue);
+        builder.addDataStrings(tagType);
         builder.addDataLongs(getTimeBucket());
         return builder;
     }
+
+    public static class Builder implements StorageBuilder<TagAutocompleteData> {
+        @Override
+        public TagAutocompleteData storage2Entity(final Convert2Entity converter) {
+            TagAutocompleteData record = new TagAutocompleteData();
+            record.setTagKey((String) converter.get(TAG_KEY));
+            record.setTagValue((String) converter.get(TAG_VALUE));
+            record.setTagType((String) converter.get(TAG_TYPE));
+            record.setTag(record.getTagKey() + "=" + record.getTagValue());
```

Comment:
Is this really useful? As well as `TraceTagAutocompleteData`?

Tag seems never used.